src/pause.sol

```diff
@@ -51,35 +51,43 @@ contract DSPause is DSAuth, DSNote {
     }
 
     // --- util ---
-    function hash(address usr, bytes memory fax, uint eta)
+    function hash(address usr, bytes32 tag, bytes memory fax, uint eta)
         internal pure
         returns (bytes32)
     {
-        return keccak256(abi.encode(usr, fax, eta));
+        return keccak256(abi.encode(usr, tag, fax, eta));
     }
 
-    // --- executions ---
-    function plot(address usr, bytes memory fax, uint eta)
+    function codehash(address usr)
```

Comment:
Are you open to overloading `hash` for this?

Or do you prefer to keep it > 4?

Reply:
I think it's a little weird since they would have quite different meanings semantically. I'm open to naming suggestions.

Reply:
I can think of:

- `code`
- `hash`
- `name`
- `sign`
- `mark`

But I don't love any of them. This function is annoying anyway tbh since for some reason solidity seems to only expose `extcodehash` in assembly.

Reply:
Do you know if they have any plans of surfacing it in Solidity? If so, it might not be worth it to rename it.

Anyway, here are a few more:

* `crux`
* `core`
* `gist`
* `pith`
* `soul`
* `quid`

These are all variations on the idea that the functions gets you the essence of the code at `usr`.

Reply:
`soul` is nice :) :pray: